netlify.toml

```diff
@@ -0,0 +1,43 @@
+# Settings in the [build] context are global and are applied to all contexts
```

Comment:
Do we need a netlify configuration? We don't do that on any of our projects -- should it be instead done within netlify itself.

Reply:
Yes, it can be done in Netlify instead.